src/main/java/com/hpe/application/automation/tools/octane/model/processors/scm/SvnSCMProcessor.java

```diff
@@ -50,104 +52,105 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.LinkedList;
 import java.util.List;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
 
 /**
  * Created by benmeior on 5/15/2016.
  */
 
-@SuppressWarnings({"squid:S2221", "squid:S00105"})
-public class SvnSCMProcessor implements SCMProcessor {
+class SvnSCMProcessor implements SCMProcessor {
```

Comment:
Isn't this and GitSCMProcessor show some duplicity in code?
Could a polymorphic approach solve some of their similarities?